pkgs/development/python-modules/materialyoucolor/default.nix

```diff
@@ -0,0 +1,59 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchurl,
+  fetchPypi,
+  setuptools,
+  pybind11,
+  requests,
+  rich,
+  pillow,
+}:
+
+let
+  test-image = fetchurl {
+    name = "test-image.jpg";
+    url = "https://unsplash.com/photos/u9tAl8WR3DI/download";
+    hash = "sha256-shGNdgOOydgGBtl/JCbTJ0AYgl+2xWvCgHBL+bEoTaE=";
+  };
```

Comment:
Is this test image free? If not, I would take it out of `checkPhase` and probably put it in `passthru.tests` independent of the derivation.

Reply:
im not sure. I am the one who requested this package, not the one who wrote the nix file. Sorry

Reply:
https://unsplash.com/photos/low-angle-photography-coconut-trees-during-golden-hour-u9tAl8WR3DI
Free to use under the [Unsplash License](https://unsplash.com/license) .